README.md

```diff
@@ -16,4 +16,7 @@ AccessibleWebDev is licensed under the MIT License - see [License](https://githu
 
 ## Gitpod
 
+Gitpod ia an open-source Kubernetes application which provides prebuilt, collaborative and development environment in your browser.  
```

Comment:
```suggestion
Gitpod is an open-source development environment in your browser.  
```

I think mentioning Kubernetes will scare new contributors away, no need to make it over complicated.